pages/linux/quotacheck.md

```diff
@@ -0,0 +1,28 @@
+# quotacheck
+
+> Scan a filesystem for disk usage; create, check and repair quota files.
+> It is best to run quota check with quotas turned off to prevent damage or loss to quota files.
+
+- Check all mounted non-NFS filesystems:
+
+`sudo quotacheck --all`
+
+- Force check even if quotas are enabled:
+
+`sudo quotacheck --force {{filesystem}}`
```

Comment:
Hmm, good point. I think most documentation I've used refers to it as file system, but I usually run the command with / or /home. So would it be more accurate to say something like {{mountpoint}} ?